entries/s/singpass.gov.sg.json

```
@@ -4,15 +4,17 @@
     "url": "https://www.singpass.gov.sg/",
     "tfa": [
       "sms",
-      "custom-software",
       "custom-hardware"
     ],
-    "documentation": "https://www.ifaq.gov.sg/SINGPASS/apps/fcd_faqmain.aspx#TOPIC_111223",
+    "custom-hardware": [
```

```suggestion
    "custom-software": [
      "Singpass app"
    ],
    "custom-hardware": [
```

They also support a six-digit passcode through the app